### ISSUE TITLE
Keypoint detection

### ISSUE THREAD

**Issue by xiaochenwise** [muelea/shapy#41]
In addition to openpose, can I use other key points to detect the model? I run the code to test my own data measured waist circumference is about 88, and my data is the thinnest girl waist circumference is 70, the fattest boy waist circumference is 115, what may be the reason

**Comment by muelea** [comment]
Yes, you can use any keypoint detector since we only use them to crop the person on the photo. The keypoints / cropping might have a small effect onto the outcome e.g. if you don't crop the full body. Measurements for a single person can be different than the ground-truth measurements. That is expected, we evaluate the average prediction on a dataset of many people.
